broadinstitute/pearl
# [JN-525] selecting site content version

This is step 2 of 3 in getting the ability to work with siteContents effectively.  This step adds the ability to select a version from the site content editor screen. This enables, e.g. going back to edit a past version.  We still do not have the ability to save changes or attach them to an environment (that will be step 3).

![image](https://github.com/broadinstitute/juniper/assets/2800795/e5cff031-7f1b-4d83-99c0-f61b3becb45f)


TO TEST:
0. repopulate ourhealth, and redeploy adminApiApp
1.  make a small change to the content in `populate/src/main/resources/seed/portals/ourhealth/siteContent/siteContent.json`
2. go to `https://localhost:3000/populate/siteContent`
3. fill in 'ourhealth' as the portal shortcode (note the filename now auto-populates)
4. press "populate"
5. go to `https://localhost:3000/ourhealth/studies/ourheart/env/sandbox/siteContent`
6. Click the 'select' button next to the `Website content version 1` 
7. confirm the dialog appears as above
8. confirm 'version 2' is available to be selected
9. selected version 2, and press 'edit version 2'
10. confirm the content updates and you can see your change
